apm-lambda-extension/extension/route_handlers.go

```diff
@@ -37,24 +37,26 @@ func handleInfoRequest(apmServerUrl string) func(w http.ResponseWriter, r *http.
 		client := &http.Client{}
 
 		req, err := http.NewRequest(r.Method, apmServerUrl, nil)
+		if err != nil {
+			log.Printf("could not create request object for %s:%s: %v", r.Method, apmServerUrl, err)
+			return
+		}
```

Comment:
Since this is happening within a handler, it would be good to inform the client that the server had an error. I think this early return will result in the client receiving a 200.